src/main/kotlin/github/GitHubImpl.kt

```diff
@@ -136,11 +136,40 @@ object GitHubImpl {
         files: Map<String, String>,
         packageIdentifier: String,
         packageVersion: String,
-        updateState: VersionUpdateState
+        updateState: VersionUpdateState,
+        terminal: Terminal
     ): GHPullRequest {
+        if (
+            data.PreviousManifestData.installerManifest?.equals(data.InstallerManifestData) == true &&
+            data.PreviousManifestData.defaultLocaleManifest?.equals(data.DefaultLocaleManifestData) == true &&
+            data.PreviousManifestData.versionManifest?.equals(data.VersionManifestData) == true
+        ) {
+            if (
+                Environment.isCI &&
+                System.getenv(Schemas.customToolEnv) != null &&
+                System.getenv(Schemas.customToolURLEnv) != null
```

Comment:
We only need to check if the environment is a CI environment because a tool may not have those environment variables set, etc